docs/04-resources/04-tutorials/hyperlane-bridge.md

```diff
@@ -0,0 +1,221 @@
+---
+sidebar_position: 300
+title: Run Hyperlane Bridge on Rootstock
+sidebar_label: Hyperlane Bridge
+tags: [Guides, hyperlane, rootstock, bridge, tutorial, rsk, blockchain, relayer, validator, wrap routes, interoperability, cross chain, defi, permissionless, mailbox, messaging]
```

Comment:
```suggestion
tags: [hyperlane, rootstock, tutorials, rsk, blockchain, interoperability, cross chain bridge, defi]
```

Reply:
Please add the article to the tutorial index

<img width="797" alt="Screenshot 2024-09-19 at 13 03 45" src="https://github.com/user-attachments/assets/a5cbcbd7-661c-4663-b8c0-73ec31da7570">